elements/storytelling/src/markdown-it-plugin/markdown-it-decorate-improved.js

```diff
@@ -89,7 +150,24 @@ function curlyInline(children, stack) {
     if (child.type === "text") lastText = child;
   });
 
+  if (stack.last.tag === "h2") {
+    const title = (lastText && lastText["content"]) || children[0].content;
+    const attrsId = (stack.last.attrs || []).find(
+      (subArr) => subArr[0] === "id"
+    )?.[1];
+    const titleSlug = title.toLowerCase().replace(/ /g, "-");
```

Comment:
Also, is there a need to include `section-` in the id? I feel like just having the slugified title would be enough (e.g. `id="what-is-lorem-ipsum"`). This would make it nicer if added to the url as well.

Reply:
First part done ✅